$OpenBSD: patch-split_cabal,v 1.1 2015/09/19 08:09:07 kili Exp $

Hackage infrastructure fuckup: according to
http://hackage.haskell.org/package/split, this needs base<4.9, but
the included cabal file still is bound to base<4.8. So from now on,
you can't rely on .cabal files shipped with distfiles but have to
check the web pages manually.

--- split.cabal.orig	Sun Apr 14 20:47:17 2013
+++ split.cabal	Tue Sep  1 07:42:02 2015
@@ -51,7 +51,7 @@ Source-repository head
 
 Library
   ghc-options:       -Wall
-  build-depends:     base <4.8
+  build-depends:     base <4.9
   exposed-modules:   Data.List.Split, Data.List.Split.Internals
   default-language:  Haskell2010
   Hs-source-dirs:    src
